src/consumption.cpp

```diff
@@ -1254,6 +1254,21 @@ void Character::modify_morale( item &food, const int nutr )
     }
 }
 
+//computes effective food volume ratio. Used when determining stomach fullness from eating.
```

Comment:
No need to repeat the name.
```suggestion
// Used when determining stomach fullness from eating.
```